src/panels/lovelace/cards/energy/hui-energy-compare-card.ts

```diff
@@ -56,14 +56,14 @@ export class HuiEnergyCompareCard
 
     return html`
       <ha-alert dismissable @alert-dismissed-clicked=${this._stopCompare}>
-        You are comparing the period
+        ${this.hass.localize("ui.panel.energy.compare.alert_info_1")}
```

Comment:
The best approach would be to have the whole sentence as one text with placehoders, but I) am not sure if there is a way to get formatted text into the translations (so we do not loose the boldness) 🤔.

Reply:
Yes, I had thought about it too but the problem is that there are two formatDates

Reply:
@zsarnett @bramkragten Is there a way to do it in one lokalize call while keeping somehow the formatting?